commons-jcs-jcache/src/main/java/org/apache/commons/jcs/jcache/JCSCachingManager.java

```diff
@@ -45,15 +46,15 @@
     private static final String DEFAULT_CONFIG =
         "jcs.default=DC\n" +
         "jcs.default.cacheattributes=org.apache.commons.jcs.engine.CompositeCacheAttributes\n" +
```

Comment:
This are the values comming from CompositeCacheAttributes. I think that we can remove all of them because they are already initialized it the CompositeCacheAttributes class.